### PR TITLE
Tyr: List users with shape

### PR DESCRIPTION
**Pull request by patochectp**

### CODE REVIEW

File: source/tyr/tyr/resources.py [hunk 1]
```diff
@@ -454,11 +469,19 @@ def map_args_to_instance(attr_name):
 
 class User(flask_restful.Resource):
     def get(self, user_id=None):
+        parser = reqparse.RequestParser()
+        parser.add_argument('disable_geojson', type=bool,
+                            default=False,
```
Comment (pbougue) [inline source/tyr/tyr/resources.py:474]: default is True (will change tests) : we don't want to output shape by default

File: source/tyr/tests/integration/users_test.py [hunk 2]
```diff
@@ -494,3 +504,49 @@ def test_deletion_keys_and_auth(create_instance, mock_rabbit):
...
     assert len(resp_auth['authorizations']) == 0
     assert mock_rabbit.called
+
+
+def test_get_user_with_shape(create_user, geojson_feature_collection):
+    """
+    We start by creating the user with a shape,
+    and we test that the attribute shape is returned and has_shape = True
+    """
+    resp = api_get('/v0/users/{}'.format(create_user))
```
Comment (pbougue) [inline source/tyr/tests/integration/users_test.py:514]: will need explicit `disable_geojson=False`

File: source/tyr/tests/integration/users_test.py [hunk 3]
```diff
@@ -494,3 +504,49 @@ def test_deletion_keys_and_auth(create_instance, mock_rabbit):
...
+
+    assert resp['has_shape'] == True
+    assert resp['shape'] == geojson_feature_collection
+
+
+def test_get_user_with_shape_and_disable_geojson_param_true(create_user, geojson_feature_collection):
```
Comment (pbougue) [inline source/tyr/tests/integration/users_test.py:520]: would be nice to test the same but not providing param, see that we default to `disable_geojson=True`

**Review by pbougue — CHANGES_REQUESTED** [review]
Looks good to me except for the default value

File: source/tyr/tyr/resources.py (hunk 1 above)
Comment (datanel) [reply source/tyr/tyr/resources.py:474]: > we don't want to output shape by default

why?

copied from **https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/interfaces/v1/Disruptions.py#L91**

Comment (datanel) [reply source/tyr/tyr/resources.py:474]: I merge and will make another PR if we change this behaviour

**Pull request merged**